.github/ISSUE_TEMPLATE/suggest_layer2.md

```diff
@@ -0,0 +1,93 @@
+---
+name: Suggest a Layer 2
+about: Suggest a Layer 2 to list on ethereum.org
+title: ""
+labels: "feature :sparkles:, content :fountain_pen:"
+assignees: ""
+---
+
+Before suggesting a layer 2, make sure you've read [our listing policy](/contributing/adding-layer-2/).
+
+Only continue with the issue if your layer 2 meets the criteria listed there.
+
+If it does, complete the following information which we need to accurately list the layer 2.
+
+**Name**
+
+<!-- Please provide the official name of the layer 2-->
+
+**Logo**
+
+<!-- Please provide a hi-res SVG or transparent PNG -->
+
+**Description**
+
+<!-- Please provide a 1-2 sentence of the layer 2-->
```

Comment:
One problem we had with our wallets was long descriptions breaking our design system ([see this PR](https://github.com/ethereum/ethereum-org-website/pull/3569)). I don't think this needs to be solved right now but just flagging it.

Reply:
Can go twitter style, X characters or less.